tests/App/config.yml

```diff
@@ -29,15 +29,10 @@ doctrine:
                 connection: default
                 mappings:
                     LiipAcme:
-                        type: php
+                        type: attribute
```

Comment:
You'll need support for annotation mapping here as well, otherwise you'll need to skip tests on PHP 7.4 builds.